client/src/app/appCtrl.js

```diff
@@ -221,19 +231,17 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       worker: false, // multithreaded, !but does NOT work with other libs in app.js or streaming
       comments: "#",
       chunk: function(chunk, parser) {
-        if (!firstChunkComplete) {
-          streamParser = parser;
```

Comment:
I guess it depends on the interpreter etc, but it seems like it's nothing compared to nothing: 
https://stackoverflow.com/questions/4043455/which-costs-more-while-looping-assignment-or-an-if-statement 
For which I'd prefer the simpler approach.